### PR TITLE
feat(helper): add helper and cui-modal component

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

* add helper services
* add cui-modal
* add cucConfig provider
* add `jsurl@^0.1.5` dependency
* add `@ovh-ux/ng-translate-async-loader@^2.0.0` peer dependency
* add `angular-translate@^2.11.0` peer dependency
* add `angular-ui-bootstrap@~1.3.3` peer dependency

ref: OM-306, OM-313, OM-315

bbeea06 — feat(helper): add helper and cui-modal component
7f2e475 — feat(vrack): add vrack service (#9)
2c4ea10 — feat(feature-availability): add feature-availability service

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

**Review by FredericEspiau — DISMISSED** [review]

File: src/helper/controller/service.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+
```
Comment (antleblanc) [inline src/helper/controller/service.js]: Extra empty line.

File: src/helper/controller/modal/service.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import assign from 'lodash/assign';
+import includes from 'lodash/includes';
+import set from 'lodash/set';
+
+
```
Comment (antleblanc) [inline src/helper/controller/modal/service.js]: Extra empty line.

File: src/helper/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    'ui.bootstrap',
+  ])
+  .service('CucControllerHelper', ControllerHelper)
+  .service('CucControllerModalHelper', ControllerModalHelper)
+  .service('CucControllerNavigationHelper', ControllerNavigationHelper)
+  .service('CucControllerRequestHelper', ControllerRequestHelper)
+  .service('CucOrderHelperService', OrderHelperService)
+  .service('CucServiceHelper', ServiceHelper)
+  .service('CucUrlHelper', UrlHelper)
+  .run(/* @ngTranslationsInject:json ./translations */);
```
Comment (antleblanc) [inline src/helper/index.js:38]: Reminder: we have to switch from XML to JSON as default format.

**Review by antleblanc — DISMISSED** [review]
LGTM except some extra empty lines.

Is there other way to prevent set JSURL as an ESLint global definition?

**Comment by antleblanc** [comment]
@cbourgois Could you please resolve existing conflicts?

**Review by FredericEspiau — DISMISSED** [review]

**Comment by cbourgois** [comment]
Rebased on `develop` + fixes

**Review by FredericEspiau — DISMISSED** [review]

**Comment by cbourgois** [comment]
Restore https://github.com/ovh-ux/ng-ovh-cloud-universe-components/commit/6746ff16d51ac6fdcb4441e0a8bc70eb2e6cb9ad

**Review by jleveugle — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

**Review by FredericEspiau — APPROVED** [review]

**Pull request merged**